test/batch/configuration.ts

```diff
@@ -69,7 +70,8 @@ describe("Configuration", () => {
 
         setup({ rules: [badRule] });
 
-        expect(await runChecks({ pr: basePR, octokit, logger, finishProcessReviews: null })).toBe("failure");
+        const api = new GitHubApi(basePR, { logger, finishProcessReviews: null, octokit });
```

Comment:
```suggestion
        const api = new GitHubApi(basePR, { logger, octokit });
```

It's not taken by `GitHubApi`. Same in other places.